### DISCUSSION TITLE
UX: remove empty/not-applicable content apps

### DISCUSSION THREAD

**Discussion by nathanwoulfe** [umbraco/Umbraco-CMS#11143] [Features and ideas]
Nothing major, just a quick UX win (IMO 😄) - when editing a document type, and that type is an element, the list-view and templates content apps are available in the header, but have no content other than a message:

![image](https://user-images.githubusercontent.com/3248070/134124105-387626ae-89ac-42da-bda6-b9838c612266.png)

If it's not applicable, why not remove it? Dead-end navigation isn't a GoodThing™️, and in this case is easily avoided.

Happy to make the change if it's deemed worthy 😃

**Comment by nul800sebastiaan** [comment]
Channeling my inner UX designer: consistency in the UI. How frustrating if you KNOW that on all other document types you could set a list view or a template. And nobody is telling you why the heck these are missing on just 2 of your document types. I'd much rather know: oh, it's not applicable... actually, maybe I didn't need this to be an element type anyway,

**Reply by nathanwoulfe** [reply]
Fair point. What about different visible state in the header (eg faded to indicate disabled/not available)?

**Reply by nul800sebastiaan** [reply]
That makes more sense to me, maybe a little help text on the "element type" toggle ("this will disable list view/template")?

@madsrasmussen what do you think?
